metadata_mapper/mappers/mapper.py

```diff
@@ -351,18 +355,22 @@ def move_date_values(self, prop, dest="temporal"):
         # 2079 times: prop=["sourceResource/spatial"]
         """
         src = prop[0].split('/')[-1]   # remove sourceResource
-        if src not in self.mapped_data:
+        src_values = self.mapped_data.get(src)
```

Comment:
this is that same issue where sometimes src is in self.mapped_data but it is actually a None value.